labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,8 +32,19 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
-  def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+  
+  val inputs = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv"
+  val outputs = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up"
+  val dict = (inputs.toList zip outputs.toList).toMap
+  
+  def translate(line: String): String = {
+    // Map sample inputs to outputs
+    line map dict
+  }
+  
+  def googleCodeJamGooglerese(lines: String*): Seq[String] = {    
+    //lines map (_ map table)
+    lines map (x => translate(x))
```

Comment:
I _think_ you can say `lines map translate` here. What you have is obviously correct, but it might be more Scala-y without the explicit anonymous function.